includes/rest/class-followers.php

```diff
@@ -115,9 +136,33 @@ public static function request_parameters() {
 			'default' => 1,
 		);
 
+		$params['per_page'] = array(
+			'type' => 'integer',
+			'default' => 10,
+		);
+
+		$params['order'] = array(
+			'type'    => 'string',
+			'default' => 'desc',
+			'enum'    => array( 'asc', 'desc' ),
+		);
+
 		$params['user_id'] = array(
 			'required' => true,
 			'type' => 'string',
+			'validate_callback' => function( $param, $request, $key ) {
+				// despite being an integer, user_id is passed as string.
+				$param = (int) $param;
+				return 0 === $param || user_can( $param, 'publish_posts' );
+			},
+		);
+
+		$params['context'] = array(
+			'type' => 'string',
+			'default' => 'outbox',
```

Comment:
`outbox`? What about `simple` and `full`? Both views (the link and the full object) are ActivityPub compatible, I will have a look at the spec maybe there is something like an output filter defined.